pkgs/misc/drivers/xboxdrv/default.nix

```diff
@@ -1,15 +1,27 @@
-{ lib, stdenv, fetchurl, sconsPackages, libX11, pkg-config
-, libusb1, boost, glib, dbus-glib }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, sconsPackages
+, libX11
+, pkg-config
+, libusb1
+, boost
+, glib
+, dbus-glib
+}:
 
 let
   version = "0.8.8";
-in stdenv.mkDerivation {
+in
+stdenv.mkDerivation {
   pname = "xboxdrv";
   inherit version;
 
-  src = fetchurl {
-    url = "https://github.com/xboxdrv/xboxdrv/archive/v${version}.tar.gz";
-    sha256 = "0jx2wqmc7602dxyj19n3h8x0cpy929h7c0h39vcc5rf0q74fh3id";
+  src = fetchFromGitHub {
+    owner = "xboxdrv";
+    repo = "xboxdrv";
```

Comment:
```suggestion
    owner = pname;
    repo = pname;
```

Reply:
i am not going to add this suggestion.
variables should only used if the values change often/are unpredictable.
if they are static leave them static. otherwise debugging is more cumbersome.

Reply:
No problem